harness/determined/common/experimental/experiment.py

```diff
@@ -274,6 +280,25 @@ def unarchive(self) -> None:
         bindings.post_UnarchiveExperiment(self._session, id=self._id)
         self.archived = False
 
+    def move_to_project(self, workspace_name: str, project_name: str) -> None:
+        """Move an experiment to a different project.
+
+        Updates both the local object and the master database with the new project and workspace.
```

Comment:
This method updates the local object and the master database with the specified project and workspace details.

Reply:
Existing text is [consistent with our style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods). Let's chat if this is something you'd like to change.

Reply:
thanks, i added a link to the google styleguide from the documentation confluence site